Chapters/2-1-Chapter8.md

```diff
@@ -0,0 +1,159 @@
+
+The cold water pierced his skin and disoriented him. For a moment, direction was irrelevant, but he remembered to hold his breath so that physics could do the rest. _Up is to the left_, he realized and then kicked his feet to set his head upright and surface. Everyone who had been dumped into the water struggled to stay afloat. He knew he didn't have the strength to help them - he had to swim.
+
+"On your back! Use your feet!" cried a maman. Thi was speaking to the kaln, but all in the group adjusted to hae direction.
+
+The cold turned into numbness as he kept his head above water and kicked his hardest toward the shore. The current was moving against him; it would surely push him back if he stopped, but his efforts were focused on getting to shore, and he had purged his mind of all other worries in order to survive. He felt a tug at his leg in the middle of an exhale and his body went under while another person clambered over his submerged back, leaving him disoriented and deprived of oxygen. He panicked and flailed around in the water until his feet touched something solid. _The water isn't so deep here_, he thought. _I must be close_. He dug his toes into the pebble bed and pushed himself back toward his goal. He made it to standing depth and stopped to catch his breath while he looked for other people make it ashore. He spotted the young maman and hae mother, struggling together quite a ways out. It looked as though the kaln's muscles were seizing in the cold and hae mother didnt have the strength to pull both of them. A surge of urgency rushed through him, and he jumped back into the depths with his arms still bound in front of him. He had swam nearly 30 paces before he reached them. The kaln was unresponsive and hae mother was dazed, trying hae best to move both of their bodies to shore, but not making much progress. Without a thought, Kipnu latched the kaln's head under his arms, making sure thi was above water. He told the maman,
+
+"I'll get hae back. Focus on yourself."
+
+Thi looked at him with glazed over eyes, confused and disoriented, but Kipnu saw that the kaln had nearly become unconscious, so he had to move.
+
+"Come on. Stay awake, kid," he said, and he pushed the water with all his legs could push. Kipnu saw hae mother follow him, as if instinct had taken over.
+
+They got ashore, and Kipnu searched for a pulse.
+
+"Shit. No no no," he said. "Don't die. You're too young for that."
+
+He quickly moved onto the ground and lifted the kaln onto his own body. Hae skin had gone cold. In a cross-armed gesture, he set his arms in front of hae chest and rapidlly pulled down at a steady pace. The others gathered around them to watch what was happening.
+
+"The fire. We have to get hae to the fire," called a man from the crowd.
+
+"That's thousands of paces away," said a woman.
+
+"Well this isn't working," the man responded.
+
+Kipnu butted in between pulses, "Everyone, gather kindling, from the forest. We'll start, a fire here."
```

Comment:
wait, they are no longer on the ship? when did they get off?

Reply:
Was it not clear that they fell into the water? I figured "struggled to stay afloat" would be enough.